server/parsec/components/memory/datamodel.py

```diff
@@ -249,10 +253,149 @@ class MemoryInvitation:
     conduit_greeter_payload: bytes | None = field(default=None, repr=False)
     conduit_claimer_payload: bytes | None = field(default=None, repr=False)
 
+    # New fields for the new invitation system
+    # TODO: remove the old fields once the new system is fully deployed
+    greeting_sessions: dict[UserID, MemoryGreetingSession] = field(default_factory=dict)
+
     @property
     def is_deleted(self) -> bool:
         return self.deleted_on is not None
 
+    @property
+    def is_completed(self) -> bool:
+        return self.deleted_reason == MemoryInvitationDeletedReason.FINISHED
```

Comment:
Do we keep old invitations for ever ?

Reply:
At the moment yes. In general, almost no information is deleted from the server.

Reply:
But we don't have a production server ATM, so that don't pose any issues